bgpd/bgp_main.c

```diff
@@ -273,6 +273,15 @@ static int bgp_vrf_new(struct vrf *vrf)
 	if (BGP_DEBUG(zebra, ZEBRA))
 		zlog_debug("VRF Created: %s(%u)", vrf->name, vrf->vrf_id);
 
+	struct interface *ifp;
```

Comment:
Can you move to the top of the function?

Reply:
Done